CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# Contribution Guide
+
+Our standards are currently *under development*.
+
+Until then, be kind and happy collaborating!
+
+[*back*](./README.md)
```

Comment:
There's another PR up for CONTRIBUTING.md, let's nuke this to avoid a merge conflict

Reply:
Affirmative